lms/djangoapps/instructor/tests/test_legacy_raw_download_csv.py

```diff
@@ -97,11 +97,11 @@ def get_expected_grade_data(
             ],
             'data': [
                 [
-                    1, u'u1', u'username', u'view@test.com', '', 0.0, 0, 0, 0, 0, 0, 0, 0, 0,
+                    1, u'u1', u'username', u'view@test.com', '', (0.0, 1.0), 0, 0, 0, 0, 0, 0, 0, 0,
                     0, 0, 0, 0.0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
                 ],
                 [
-                    2, u'u2', u'username', u'view2@test.com', '', 0.3333333333333333, 0, 0, 0,
+                    2, u'u2', u'username', u'view2@test.com', '', (1.0, 3.0), 0, 0, 0,
                     0, 0, 0, 0, 0, 0, 0, 0, 0.03333333333333333, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
I was puzzled why I had to replace the first non-zero grade for this user but not the second.  I found that the second non-zero score is the average grade for the category (``HW Avg``) so the data for points earned and max points possible is not easily available.  @pdpinch